client/mass/nav.js

```diff
@@ -224,6 +226,12 @@ function setRenderers(self) {
 			titleDiv,
 			tip: new Menu({ padding: '5px' })
 		}
+		const deleteAllDiv = controlsDiv
+			.append('div')
+			.style('display', 'inline-block')
+			.style('padding-top', '4px')
+			.style('vertical-align', 'middle')
+		icon_functions['trash'](deleteAllDiv, { handler: self.deletePlots, title: 'Delete all' })
```

Comment:
It's not clear that this applies to plots/charts/graphs, should title as `Delete all plots`. Otherwise, it may be interpreted as delete all filters or cohorts when those tabs are open.

Reply:
yeah, just profile calls the plots graphs, I was trying to use something that works for all